packages/grafana-e2e-selectors/src/selectors/pages.ts

```diff
@@ -138,11 +138,13 @@ export const Pages = {
             generalHideSelect: 'Variable editor Form Hide select',
             generalHideSelectV2: 'data-testid Variable editor Form Hide select',
             selectionOptionsMultiSwitch: 'Variable editor Form Multi switch',
+            selectionOptionsMultiSwitchV2: 'data-testid Variable editor Form Multi switch',
```

Comment:
te parece si eliminamos este selector, y solo agregamos el data-testid en el `selectionOptionsMultiSwitch`?

Reply:
Done! I forgot about that one!